### ISSUE TITLE
Introduce `--format` flag to fine control over different report formats

### ISSUE THREAD

**Issue by Dentrax** [google/osv-scanner#217]
To get fine control over different formats and to make it more sustainable in the long term.

Here is simple idea:

```
-f, --format string          format (table, json, sarif, etc...) (default "table")
```

Deprecate the `--json` flag.

Also, @G-Rath have some thoughts on this idea: https://github.com/google/osv-scanner/issues/117#issuecomment-1416149996

**Comment by marcinguy** [comment]
@Dentrax You can get JSON, SARIF, HTML, CLI output and scanning GH action using betterscan https://github.com/marcinguy/betterscan-ce

It has OSV scanner as one the scanners

Maybe it hells

**Comment by G-Rath** [comment]
@Dentrax this has already been implemented in #158

**Comment by Dentrax** [comment]
Oh 🙈 How I missed this! I just installed the tool with `brew` and didn't see `--format` option in the flags. Probably, it hasn't been released yet. Thanks - will check this!
